slack.json

```diff
@@ -1864,6 +1864,11 @@
                     "description": "Used in conjunction with 'parent_message_ts' and indicates whether reply should be made visible to everyone in the channel or conversation",
                     "data_type": "boolean",
                     "order": 3
+                },
+                "link_names": {
+                    "description": "Link names added in message as <!group>. e.g. @group will notify the group.",
```

Comment:
Can you please provide a more in depth description if possible? Seems a bit vague